pypeit/flatfield.py

```diff
@@ -1946,6 +1947,76 @@ def illum_profile_spectral(rawimg, waveimg, slits, slit_illum_ref_idx=0, smooth_
     return scaleImg
 
 
+def illum_profile_spectral_poly(rawimg, waveimg, slits, model, slit_illum_ref_idx=0,
+                                gpmask=None, thismask=None, trim=3, flexure=None, nbins=20):
+    """
+    Use a polynomial fit to control points along the spectral direction to determine the relative
+    spectral illumination of all slits. Currently, this routine is only used for image slicer IFUs.
+
+    Parameters
+    ----------
+    rawimg : `numpy.ndarray`_
+        Image data that will be used to estimate the spectral relative sensitivity
+    waveimg : `numpy.ndarray`_
+        Wavelength image
+    slits : :class:`~pypeit.slittrace.SlitTraceSet`
```

Comment:
Slits is an object. Instead pass in the boundaries, slit_left and slit_right which are arrays. That is our convention in core.

Reply:
Okay my bad. Can you move it to core and wrap here?

Reply:
Yep - I've already moved it to core  :-)
(note, this file is marked "outdated")